### PR TITLE
EIA Electricity: add full schema support for non-plant series 

### PR DESCRIPTION
**Pull request by pradh**
Validation details included in the manifest CL.

### CODE REVIEW

File: scripts/us_eia/opendata/process/elec.py [hunk 1]
```diff
@@ -101,43 +102,144 @@ def extract_place_statvar(series_id, stats):
...
+# units (see _UNIT_MAP).
+_MEASURE_MAP['COST_BTU'] = _MEASURE_MAP['COST']
+_MEASURE_MAP['CONS_EG_BTU'] = _MEASURE_MAP['CONS_EG']
+_MEASURE_MAP['CONS_TOT_BTU'] = _MEASURE_MAP['CONS_TOT']
+_MEASURE_MAP['CONS_UTO_BTU'] = _MEASURE_MAP['CONS_UTO']
+_MEASURE_MAP['RECEIPTS_BTU'] = _MEASURE_MAP['RECEIPTS']
+
+
+# Unit with this value is handled specially depending on the fuel type.
+_FUEL_UNIT = '_XYZ_'
```
Comment (beets) [inline scripts/us_eia/opendata/process/elec.py:208]: maybe _PLACEHOLDER_FUEL_UNIT or _TMP_FUEL_UNIT?

**Review by beets — APPROVED** [review]
thanks!

File: scripts/us_eia/opendata/process/common.py [hunk 2]
```diff
@@ -156,13 +156,21 @@ def process(in_json, out_csv, out_sv_mcf, out_tmcf, extract_place_statvar_fn,
...
                     continue
 
+                # TODO: Consider extracting units.
+
                 # Add to rows.
                 rows = []
                 for k, v in time_series:
+
+                    if not v:
+                        stats['error_empty_values'] += 1
```
[the remark this answers is not in the document]
Comment (pradh) [reply scripts/us_eia/opendata/process/common.py]: good point, renamed all "stats" refs to "counters"

File: scripts/us_eia/opendata/process/elec.py (hunk 1 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/elec.py]: Done.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**